frontend/src/pages/ConnectedPages/Playlists/components/PlaylistCard/index.tsx

```diff
@@ -3,20 +3,57 @@ import React, { useState } from "react";
 import { IPlaylist } from "../../types";
 import "./playlistCard.css";
 import { Redirect } from "react-router-dom";
+import { axiosRequest } from "../../../components/axiosRequest";
+import { Method } from "axios";
+import { IRoom } from "../../../components/types";
 
 interface IProps {
   playlist: IPlaylist;
 }
 
+// BEGIN TODO place that in a wrapper for components to easily access
+const ping = (ws: WebSocket, gameId: number): void => {
+  console.log("ping");
+  ws.send(JSON.stringify({ type: "PING", gameId }));
+  setTimeout(() => ping(ws, gameId), 1000);
+};
+
+const testWebsocket = (gameId: number): void => {
```

Comment:
Since it going to be the final function, should we name it with its final name now ?

Reply:
This will not be the final function. It still needs to be extracted and made available to other components (we will have only one websocket client for the whole front end). 
This was to test end to end and show how it will be done